### DISCUSSION TITLE
Comparing file with local copy not working

### DISCUSSION THREAD

**Discussion by shauryagoel** [jesseduffield/lazygit#3882] [General]
I added the [following](https://github.com/jesseduffield/lazygit/wiki/Custom-Commands-Compendium#comparing-a-file-in-a-previous-revision-with-the-working-copy) custom command. But, on running it nothing is happening except the command being displayed in the command logs. Has anybody been able to make it work?

**Comment by mark2185** [comment]
Does running the command manually do anything?

**Reply by shauryagoel** [reply]
Yes, the command is running without lazygit.
